src/annotator/integrations/pdf.js

```diff
@@ -115,7 +147,7 @@ export class PDFIntegration {
     }
 
     try {
-      const hasText = await pdfAnchoring.documentHasText();
+      const hasText = await documentHasText();
       this._showNoSelectableTextWarning(!hasText);
```

Comment:
Unrelated with this PR, but this is unusual. Why not?
```suggestion
      if (!hasText) {
          this._showNoSelectableTextWarning();
      }
```

Why do we want to create the `<hypothesis-banner>` with its shadow DOM for nothing?

Reply:
Ah - the method name is confusing here. The way it behaves is more like `this._toggleSelectableTextWarning(<visible>)`. If the value is `false` it just removes the banner if it already exists.